### PR TITLE
Add node 4 on travis

### PR DESCRIPTION
**Pull request by Swiip**
Just for fun :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ sudo: false
 node_js:
   - '0.10'
   - '0.12'
+  - '4.0'
```
Comment (zckrs) [inline .travis.yml]: replace by stable

**Comment by busches** [comment]
If the build is already passing in `stable`, should it really be added to `allow_failures`?

**Comment by zckrs** [comment]
Already passing with error when install phantomjs

https://travis-ci.org/Swiip/generator-gulp-angular/jobs/80208134#L501

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ sudo: false
 node_js:
   - '0.10'
   - '0.12'
+  - stable
   - iojs
```
Comment (zckrs) [inline .travis.yml]: Remove iojs ?

**Comment by zckrs** [comment]
Build with node v4 stable fail because socket.io must update depencie engine.io to 1.5.4

https://github.com/socketio/socket.io/issues/2228
https://github.com/socketio/socket.io/issues/2242
https://github.com/socketio/socket.io/issues/2245
https://github.com/socketio/socket.io/issues/2213

**Comment by Swiip** [comment]
Done

**Pull request merged**